browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -3819,4 +3819,195 @@ TEST_F(KeyringServiceAccountDiscoveryUnitTest, RestoreWalletTwice) {
   EXPECT_THAT(requested_addresses, ElementsAreArray(&saved_addresses()[1], 30));
 }
 
+class KeyringServicePasswordIternationsMigrationUnitTest
+    : public KeyringServiceUnitTest {
+ public:
+  KeyringServicePasswordIternationsMigrationUnitTest() {
+    feature_list_.InitWithFeatures(
+        {brave_wallet::features::kBraveWalletFilecoinFeature,
+         brave_wallet::features::kBraveWalletSolanaFeature},
+        {});
+  }
+
+  void SetupKeyring() {
+    KeyringService service(json_rpc_service(), GetPrefs());
+    saved_mnemonic_ = *CreateWallet(&service, "brave");
```

Comment:
we also need `RestoreWallet`, not just `CreateWallet`

Reply:
Added test for RestoreWallet without initial setup here https://github.com/brave/brave-core/pull/14523/commits/bd0e5d57772786a7be7a3b7e51d08caf9c16ea23